### DISCUSSION TITLE
How to use translations in form validation message v3?

### DISCUSSION THREAD

**Discussion by undjike** [livewire/livewire#6357] [1. Help]
I don't know if I missed the tip. But how can I use translations in the form validation message?
Ex: __('error.message')

![image](https://github.com/livewire/livewire/assets/57102409/74606b29-4807-411e-b840-e62428494497)

Or I should use the traditionnal syntax for that.

**Comment by PerryvanderMeer** [comment]
Currently, there is a related PR: https://github.com/livewire/livewire/pull/6344

**Reply by undjike** [reply]
It's for the `as` attribute. Hopefully, `message` will be taken into consideration too.

**Reply by theChaosCoder** [reply]
Faced the same issue. No easy way to translate message.

**Reply by joshhanley** [reply]
@undjike and @theChaosCoder PR has been merged and it includes support for translated messages too 🙂

**Reply by undjike** [reply]
Thanks 🎊🙏
